File: website/src/models/OrderFlowerDTO.js
export default class OrderFlowerDTO {
    constructor(data) {
        this.orderId = data && data.orderId ? data.orderId : " ";
        this.flowerId = data && data.flowerId ? data.flowerId : " ";
        this.quantity = data && data.quantity ? data.quantity : 0;
    }
}

File: src/main/java/com/labwork01/app/order/model/OrderFlowerDTO.java
package com.labwork01.app.order.model;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class OrderFlowerDTO {
    private long orderId;
    private long flowerId;
    private int quantity;
    public OrderFlowerDTO(){

    }
    public OrderFlowerDTO(OrderFlower orderFlower){
        this.orderId = orderFlower.getOrder().getId();
        this.flowerId = orderFlower.getFlower().getId();
        this.quantity = orderFlower.getQuantity();
    }
    public long getOrderId(){
        return orderId;
    }
    public long getFlowerId(){ return flowerId; }
    public int getQuantity(){
        return quantity;
    }
}
